exo-go/src/application/docker_compose_builder.go

```diff
@@ -14,18 +14,31 @@ import (
 // DockerComposeBuilder returns the given service config that will appear
 // in docker-compose.yml
 type DockerComposeBuilder struct {
-	AppConfig     types.AppConfig
-	ServiceConfig types.ServiceConfig
-	ServiceData   types.ServiceData
-	Role          string
-	AppDir        string
-	HomeDir       string
+	AppConfig                types.AppConfig
+	ServiceConfig            types.ServiceConfig
+	ServiceData              types.ServiceData
+	BuiltAppDependencies     map[string]config.AppDependency
```

Comment:
I don't think we need `Built` in this property name. Same for `BuiltServiceDependencies`. Applies throughout this PR.